processes/releases.md

```diff
@@ -0,0 +1,42 @@
+# New MeiliSearch releases - what to do
+
+Those are the steps we need to take every time we release a new version of MeiliSearch.
+
+## Responsibilities
+
+Right now, unless mentioned, the tasks for a new release on the developer relations side are divided in three categories or actions with each person accountable for them:
+- Documentation & guides: [@react-learner](https://github.com/react-learner)
+- Blog posts: [@CaroFG](https://github.com/CaroFG)
+- Social media: [@fharper](https://github.com/fharper)
+
+## After the last release
+
+1. Create a PR on a branch called prepare-meilisearch-[YYYY]-[WW] (i.e.: prepare-meilisearch-2020-03 for the third week of the year 2020) in the [documentation repository](https://github.com/meilisearch/documentation).
+1. Ensure that all new PRs related to the next release are done on the mentioned above.
```

Comment:
I did this by adding every PR from the MeiliSearch Milestone in the body of the prepare-meilisearch-[YYYY]-[WW] body like this: 

<img width="671" alt="Screenshot 2021-01-06 at 09 42 07" src="https://user-images.githubusercontent.com/33010418/103747791-79971300-5003-11eb-8330-293a6c8dbf0c.png">

When I overlined a PR it meant that it did not need to be documented. When it did, I added a checkbox in front that I only checked once the documentation for this PR was done, reviewed, and merged in the prepare-meilisearch-[YYYY]-[WW] branch. 
That way, if any was to be forgotten, we would know very quickly

Reply:
This sentence is not clear for me; I understand it thanks to Charlotte's example.

Reply:
I updated the text and steps. Open another conversation if needed. thanks for the suggestion!